### PR TITLE
Create npm Schemas Library for easy schema and parquet creation access

### PR DESCRIPTION
**Pull request by wilwade**
Problem
=======
As a developer, I want to be able to easily interact with the schemas and create parquet files from the schemas

Solution
========
Lots of stuff, but in general getting it so it can be an npm package.

Change summary:
---------------
* Upgraded to the latest polkadotjs
* Deployment is now always in the same order
* Moved the deploy and read scripts into a cli directory
* Added package build and release logic
* Added a parquet helper that can build the correct parquet files `fromFrequencySchema`

Steps to Verify (Library):
----------------
1. Clone
1. npm i
1. npm run build
1. cd dist && npm pack
1. Create a different npm package and follow the instructions on the readme using the packed tgz file.

Steps to Verify (Deploy):
----------------
1. Run Frequency in instant sealing mode
2. npm run deploy
3. npm run read

### CODE REVIEW

File: cli/deploy.ts [hunk 1]
```diff
@@ -54,27 +53,30 @@ const eventWithSectionAndMethod = (events: EventRecord[], section: string, metho
...
     process.exit(1);
   }
 
   // Retrieve the current account nonce so we can increment it when submitting transactions
-  let nonce = (await api.rpc.system.accountNextIndex(signerAccountKeys.address)).toNumber();
+  const baseNonce = (await api.rpc.system.accountNextIndex(signerAccountKeys.address)).toNumber();
+
+  for (const idx in schemaNames) {
+    const schemaName = schemaNames[idx];
+    const nonce = baseNonce + Number(idx);
```
Comment (wilwade) [inline cli/deploy.ts:74]: The base nonce and this ensure that the schemas are always deployed in the same order.

File: cli/services/connect.ts
Comment (wilwade) [inline cli/services/connect.ts:1]: Cleanup for the new version of `@polkadot/api`

File: dsnp/index.ts [hunk 2]
```diff
@@ -9,68 +11,92 @@ import publicKey from "./publicKey";
...
+export type Deploy = ParquetDeploy | AvroDeploy;
+
+export type ParquetSchemaName = "broadcast" | "profile" | "reaction" | "reply" | "tombstone" | "update";
+export type AvroSchemaName = "publicKey" | "userPublicFollows" | "userPrivateFollows" | "userPrivateConnections";
+
+export type SchemaName = ParquetSchemaName | AvroSchemaName;
+
 // Map schema names (string) to deploy object
-export const dsnpSchemas = new Map<string, Deploy>([
+export const schemas = new Map<SchemaName, Deploy>([
```
Comment (wilwade) [inline dsnp/index.ts:53]: Reordered to align with mainnet Frequency

File: types/frequency.ts [hunk 3]
```diff
@@ -1,18 +1,21 @@
...
   column_type: FrequencyParquetType;
   compression: ColumnCompressionCodec;
   bloom_filter: boolean;
 }
 
-type ColumnCompressionCodec = "UNCOMPRESSED" | "SNAPPY" | "GZIP" | "LZO" | "BROTLI" | "ZSTD" | "LZ4RAW";
+type ColumnCompressionCodec = ParquetCompression;
 
-type ParquetBaseType = "BOOLEAN" | "INT32" | "INT64" | "FLOAT" | "DOUBLE" | "BYTE_ARRAY" | "FIXED_LENGTH_BYTE_ARRAY";
+type ParquetBaseType = "BOOLEAN" | "INT32" | "INT64" | "FLOAT" | "DOUBLE" | "BYTE_ARRAY" | "FIXED_LEN_BYTE_ARRAY";
```
Comment (wilwade) [inline types/frequency.ts:18]: Bug fix. This is wrong in Frequency as well and will have a PR to fix it soon

**Review by shannonwells — APPROVED** [review]
Nice table output on the npm read.  Looks great!

**Pull request merged**